cas-server-webapp/src/main/webapp/WEB-INF/unused-spring-configuration/clearpass-configuration.xml

```diff
@@ -83,14 +83,24 @@
    -->
     <!--
       NOTE:
-      A bean named clearPassProxyList must be defined in deployerConfigContext.xml that defines
+      A bean named clearPassProxyList must be defined that defines
       the list of proxying services authorized to obtain clearpass credential.
     -->
+    <bean id="clearPassProxyList" class="org.jasig.cas.client.validation.ProxyList">
+        <constructor-arg>
+            <list>
+                <value>...</value>
+            </list>
+        </constructor-arg>
+    </bean>
+
     <bean id="casValidationFilter" class="org.jasig.cas.client.validation.Cas20ProxyReceivingTicketValidationFilter"
           p:serverName="${server.name}" p:exceptionOnValidationFailure="false"
           p:useSession="true" p:ticketValidator-ref="clearPassTicketValidator"/>
 
     <bean id="clearPassTicketValidator" class="org.jasig.cas.client.validation.Cas20ProxyTicketValidator"
+          p:allowEmptyProxyChain="false"

```

Comment:
That's fine idea. I'll make the change.

Note that I am not sure it applies to clearpass. For extra security, it's best to enable clearpass+proxying via disabling empty proxy chains and not accepting any proxy. Otherwise, this could result into a security bypass issue where a service without having gone through the proxy dance gets to access clearpass. So, we want this value to always remains false. No need to further customize, in most cases.